src/components/Disclaimer.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Modal from '@material-ui/core/Modal';
+
+
+function getModalStyle() {
+  const top = 50;
+  const left = 50;
+
+  return {
+    top: `${top}%`,
+    left: `${left}%`,
+    transform: `translate(-${top}%, -${left}%)`,
+  };
+}
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    position: 'absolute',
+    width: 600,
+    backgroundColor: theme.palette.background.paper,
+    border: '2px solid #000',
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+    [theme.breakpoints.down('xs')]: {
+      width: '90vw',
+      padding: theme.spacing(2, 2)
+    }
+  },
+  disclaimerButton: {
+    width: '100vw',
+    display: 'flex',
+    justifyContent: 'center',
+    padding: '5px',
+  },
+  disclaimerText: {
+    [theme.breakpoints.down('xs')]: {
+      fontSize: '10px',
+    }
+  }
+}));
+
+export default function SimpleModal() {
```

Comment:
@katiemoon1 can we rename this to `Disclaimer()` to match the filename? Will need updated in `StickyFooter.js` as well.

Reply:
Done!